File: tests/fixtures/sample.java
package com.example;

import java.util.List;

public class Calculator {
    public int add(int a, int b) {
        return a + b;
    }

    public int multiply(int a, int b) {
        return a * b;
    }
}

class Main {
    public static void main(String[] args) {
        Calculator calc = new Calculator();
        int result = calc.add(1, 2);
        System.out.println(result);
    }

    static final int DEFAULT = 0;
}

File: tests/fixtures/sample.py
"""Sample Python file for extractor testing."""


class Calculator:
    """A simple calculator class."""

    def add(self, a: int, b: int) -> int:
        return a + b

    def multiply(self, a: int, b: int) -> int:
        return a * b


def greet(name: str) -> str:
    """Greet someone by name."""
    return f"Hello, {name}"


def main():
    calc = Calculator()
    result = calc.add(1, 2)
    greeting = greet("world")
    print(greeting)
    print(result)


COUNT: int = 0
